packages/nameguard-sdk/src/index.test.ts

```diff
@@ -96,7 +96,7 @@ describe("NameGuard", () => {
     );
 
     expect(data.primary_name_status).toBe("normalized");
-    expect(data.impersonation_status).toBe("potential");
+    expect(data.impersonation_estimate).toBe("potential");
```

Comment:
I see a "Check failure" on this line inside GitHub's PR review interface. Could you please investigate? Thanks